### PR TITLE
Pre phishing alerts

### PR DESCRIPTION
**Pull request by tomer-pan**
## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
This PR will add changes as part of the phishing v3 efforts. The changes in this PR are necessary in order to move forward with the "phishing alerts" efforts.

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by richardbluestone** [comment]
Reviewed and updated (after the previous review was restricted)

File: Packs/Gmail/Playbooks/playbook-Get_Original_Email_-_Gmail_v2_README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+* gmail-search
+* gmail-get-attachments
+
+## Playbook Inputs
+---
+
+| **Name** | **Description** | **Default Value** | **Required** |
+| --- | --- | --- | --- |
+| MessageID | The original email message id to retrieve. Holds the value of the "Message-ID" header of the original email and not Gmail's internal ID of the message. |  | Optional |
+| UserID | The user's email address. The "me" special value can be used to indicate the authenticated user. |  | Optional |
```
Comment (David-BMS) [inline Packs/Gmail/Playbooks/playbook-Get_Original_Email_-_Gmail_v2_README.md:31]: Improve the description here

File: Packs/Gmail/Playbooks/playbook-Get_Original_Email_-_Gmail_v2.yml [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+  value: {}
+  required: false
+  description: The original email message id to retrieve. Holds the value
+    of the "Message-ID" header of the original email and not Gmail's internal ID of
+    the message.
+  playbookInputQuery:
+- key: UserID
+  value: {}
+  required: false
+  description: The user's email address. The "me" special value can be used to indicate
```
Comment (David-BMS) [inline Packs/Gmail/Playbooks/playbook-Get_Original_Email_-_Gmail_v2.yml:344]: Improve the description here

File: Packs/Phishing/Playbooks/Phishing_Investigation_-_Generic_v2_-_6_0.yml [hunk 3]
```diff
@@ -179,19 +179,21 @@ tasks:
...
+      append: {}
       key:
         simple: ReporterAddress
+      stringify: {}
       value:
         complex:
-          root: incident
-          accessor: labels.Email/from
+          root: ExtractedIndicators.Email
+          accessor: '[0]'
```
Comment (David-BMS) [inline Packs/Phishing/Playbooks/Phishing_Investigation_-_Generic_v2_-_6_0.yml:189]: is the [0] intentional?

File: Packs/Phishing/Playbooks/Process_Email_-_Generic_v2_README.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+| UserID | The user's email address for which to retrieve the original email. This value will be passed as an input to the playbook "Get Original Email - Generic v2". | incident.emailfrom | Optional |
+| Thread-Topic | The value of the "Thread-Topic" header which holds the original email subject. This is necessary for forwarded emails scenarios. It will be passed as an input to the "Get Original Email - Generic v2" playbook to be used in the relevant sub-playbooks. |  | Optional |
+
+## Playbook Outputs
+---
+
+| **Path** | **Description** | **Type** |
+| --- | --- | --- |
+| Email.HTML | Email 'html' body if exists. | string |
+| Email | Email object. | unknown |
```
Comment (David-BMS) [inline Packs/Phishing/Playbooks/Process_Email_-_Generic_v2_README.md:54]: Change to string as well

File: Packs/Phishing/Playbooks/Process_Email_-_Generic_v2_README.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+| Email.CC | Email 'cc' addresses. | string |
+| Email.From | Email 'from' sender. | string |
+| Email.Subject | Email subject. | string |
+| Email.To | Email 'to' addresses. | string |
+| Email.Text | Email 'text' body if exists. | string |
+| Email.Headers | The full email headers as a single string. | string |
+| Email.Attachments | The list of attachment names in the email. | string |
+| Email.Format | The format of the email if available. | string |
+| File | The File object. | unknown |
```
Comment (David-BMS) [inline Packs/Phishing/Playbooks/Process_Email_-_Generic_v2_README.md:63]: Change to string as well

File: Packs/Phishing/Playbooks/playbook-Get_Original_Email_-_Generic_v2.yml [hunk 6]
```diff
@@ -0,0 +1,408 @@
...
+    - Gmail
+    - EWS v2
+    - MicrosoftGraphMail
+
+    If none of the above values is supplied, all of the playbooks will run.
+  playbookInputQuery:
+outputs:
+- contextPath: Email
+  description: The email object
+  type: unknown
```
Comment (David-BMS) [inline Packs/Phishing/Playbooks/playbook-Get_Original_Email_-_Generic_v2.yml:372]: Change to string as well

File: Packs/Phishing/Playbooks/playbook-Get_Original_Email_-_Generic_v2.yml [hunk 7]
```diff
@@ -0,0 +1,408 @@
...
+  type: string
+- contextPath: Email.HTML
+  description: The email HTML
+  type: string
+- contextPath: Email.Body
+  description: The email text body
+  type: string
+- contextPath: Email.Headers
+  description: The email headers
+  type: unknown
```
Comment (David-BMS) [inline Packs/Phishing/Playbooks/playbook-Get_Original_Email_-_Generic_v2.yml:396]: Change to string as well

File: Packs/Phishing/Playbooks/playbook-Get_Original_Email_-_Generic_v2_README.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+| UserID | The user's email address. The "me" special value can be used to indicate the authenticated user \(gmail\). |  | Optional |
+| EmailSubject | The original email subject. |  | Optional |
+| EmailBrand | When this value is supplied only the relevant playbook will run.<br/>Possible values:<br/>- Gmail<br/>- EWS v2<br/>- MicrosoftGraphMail<br/><br/>If none of the above values is supplied, all of the playbooks will run. |  | Optional |
+
+## Playbook Outputs
+---
+
+| **Path** | **Description** | **Type** |
+| --- | --- | --- |
+| Email | The email object | unknown |
```
Comment (David-BMS) [inline Packs/Phishing/Playbooks/playbook-Get_Original_Email_-_Generic_v2_README.md:45]: Change to string as well

File: Packs/Phishing/Playbooks/playbook-Get_Original_Email_-_Generic_v2_README.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+| File | Original attachments | unknown |
```
Comment (David-BMS) [inline Packs/Phishing/Playbooks/playbook-Get_Original_Email_-_Generic_v2_README.md:46]: Change to string as well

File: Packs/Phishing/Playbooks/playbook-Get_Original_Email_-_Generic_v2_README.md [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+| Email.To | The recipient of the email | string |
+| Email.From | The sender of the email | string |
+| Email.CC | The CC address of the email | string |
+| Email.BCC | The BCC address of the email | string |
+| Email.HTML | The email HTML | string |
+| Email.Body | The email text body | string |
+| Email.Headers | The email headers | unknown |
+| Email.Subject | The email subject | string |
+| Email.HeadersMap | The headers of the email. | unknown |
```
Comment (David-BMS) [inline Packs/Phishing/Playbooks/playbook-Get_Original_Email_-_Generic_v2_README.md:55]: Change to string as well

File: Packs/Phishing/Playbooks/playbook-Get_Original_Email_-_Generic_v2_README.md [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+| reportedemailentryid | In case the original eml was retrieved, this field will hold the File's Entry ID. | unknown |
```
Comment (David-BMS) [inline Packs/Phishing/Playbooks/playbook-Get_Original_Email_-_Generic_v2_README.md:56]: Change to string as well

**Review by David-BMS — APPROVED** [review]

File: Packs/Phishing/Playbooks/Phishing_Investigation_-_Generic_v2_-_6_0.yml (hunk 3 above)
Comment (tomer-pan) [reply Packs/Phishing/Playbooks/Phishing_Investigation_-_Generic_v2_-_6_0.yml:189]: Yes. The extracted email address will be always the first item.

**Pull request merged**